layers/exporters/to_svg.py

```diff
@@ -8,8 +10,278 @@
 class NoLayer(Exception):
     pass
 
+
+class SVGConfig:
+    def __init__(self, width=8.5, height=11, headerHeight=1, unit="in", showSubtechniques="expanded",
+                 font="sans-serif", tableBorderColor="#6B7279", showHeader=True, legendDocked=True,
+                 legendX=0, legendY=0, legendWidth=2, legendHeight=1, showLegend=True, showFilters=True,
+                 showAbout=True, border=0.104):
+        """
+            Define parameters to configure SVG export
+
+            :param width: Desired SVG width
+            :param height: Desired SVG height
+            :param headerHeight: Desired Header Block height
+            :param unit: SVG measurement units (qualifies width, height, etc.)
+            :param showSubtechniques: Display form for subtechniques - "all", "expanded" (decided by layer), or "none"
+            :param font: What font style to use - "sans", "sans-serif", or "monospace"
+            :param tableBorderColor: Hex color to use for the technique borders
+            :param showHeader: Whether or not to show Header Blocks
+            :param legendDocked: Whether or not the legend should be docked
+            :param legendX: Where to place the legend on the x axis if not docked
+            :param legendY: Where to place the legend on the y axis if not docked
+            :param legendWidth: Width of the legend if not docked
+            :param legendHeight: Height of the legend if not docked
+            :param showLegend: Whether or not to show the legend
+            :param showFilters: Whether or not to show the Filter Header Block
+            :param showAbout: Whether or not to show the About Header Block
+            :param border: What default border width to use
+        """
+        self.width = width
+        self.height = height
+        self.headerHeight = headerHeight
+        self.unit = unit
+        self.showSubtechniques = showSubtechniques
+        self.font = font
+        self.tableBorderColor = tableBorderColor
+        self.showHeader = showHeader
+        self.legendDocked = legendDocked
+        self.legendX = legendX
+        self.legendY = legendY
+        self.legendWidth = legendWidth
+        self.legendHeight = legendHeight
+        self.showLegend = showLegend
+        self.showFilters = showFilters
+        self.showAbout = showAbout
+        self.border = border
+
+    def load_from_file(self, filename=""):
+        """
+            Load config from a json file
+
+            :param filename: The file to read from
+        """
+        with open(filename, 'r') as fio:
+            raw = fio.read()
+            self._data = json.loads(raw)
+        for entry in self._data:
+            if entry in vars(self).keys():
+                setattr(self, entry, self._data[entry])
+            else:
+                print('WARNING - Unidentified Config Field in {}: {}'.format(filename, entry))
```

Comment:
If I understand how things work correctly, there's already going to be validation, since setting the attributes will be redirected through the setter functions. I just tried setting the font to "times-new-roman", and it blocked the set, as expected.